pkg/management/component.go

```diff
@@ -272,6 +272,23 @@ func (c *component) UpdateUser(ctx context.Context, realmName, userID string, us
 
 	userRep = api.ConvertToKCUser(user)
 
+	// Merge attributes coming from old user and updated one to not loose anything
```

Comment:
Merge the attributes coming from the old user representation and the updated user representation in order not to lose anything

Reply:
This is a suggestion as your comment is a bit fuzzy

Reply:
Maybe you can also add tests with having empty attributes in the updated user representation ?

Reply:
Thanks for the improved comments.
Good idea for the unittests, I did it.